### PR TITLE
ROX-24509: Add filter telemetry Node/Platform CVEs

### PR DESCRIPTION
**Pull request by dvail**
### Description

As titled, based on the equivalent change in Workload CVEs. https://github.com/stackrox/stackrox/pull/11747

### User-facing documentation

- [ ] CHANGELOG update is not needed
- [ ] Documentation is not needed

### Testing

- [ ] inspected CI results

#### Automated testing

- [ ] contributed **no automated tests**

_Will add automated tests for telemetry in the future._

#### How I validated my change

Follow the manual verification process described in https://github.com/stackrox/stackrox/pull/11747 for the following pages:

- Node CVE Overview
- Node single page
- Node CVE single page
- Platform CVEs Overview
- Cluster single page
- Platform CVE single page

### CODE REVIEW

File: ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Node/useNodeVulnerabilities.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default function useNodeVulnerabilities({
 }: { nodeId: string; query: string } & ClientPagination) {
     return useQuery<
         {
-            node: {
+            node?: {
```
Comment (dvail) [inline ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Node/useNodeVulnerabilities.ts:29]: An unrelated issue I found when I accidentally had an API response mocked in local dev. This type can techincally return `null`, which if not accounted for will cause page crashes.

**Comment by dvail** [comment]
ocp-4-15 failure to create cluster

File: ui/apps/platform/src/hooks/useAnalytics.ts [hunk 2]
```diff
@@ -132,13 +134,16 @@ export type AnalyticsEvent =
...
+     * This is controlled by the main search bar on all VM CVE pages.
      * We only track the value of the applied filter when it does not represent
      * specifics of a customer environment.
      */
     | {
-          event: typeof WORKLOAD_CVE_FILTER_APPLIED;
+          event:
+              | typeof WORKLOAD_CVE_FILTER_APPLIED
+              | typeof NODE_CVE_FILTER_APPLIED
+              | typeof PLATFORM_CVE_FILTER_APPLIED;
```
Comment (sachaudh) [inline ui/apps/platform/src/hooks/useAnalytics.ts:146]: I was looking at this file and feel we should separate the constants and types into separate files for readability purposes. We don't need to do anything right now, but we should consider it as the file gets larger.

**Review by sachaudh — APPROVED** [review]
LGTM 🚢

File: ui/apps/platform/src/hooks/useAnalytics.ts (hunk 2 above)
Comment (dvail) [reply ui/apps/platform/src/hooks/useAnalytics.ts:146]: Yes I agree. Maybe something like we are planning for the search filter configs? Each `Container` has its own set of exports that it defines?

**Pull request merged**